### ISSUE TITLE
Imagenet Example: Training loss goes to 0 immediately

### ISSUE THREAD

**Issue by SBoulanger** [pytorch/examples#737]
This is a very weird bug... I am trying to train on ImageNet, but when I run the script
`python3 main.py -a alexnet -j 10 --epochs 100 -b 32 --lr 0.01 /data/ImageNet/ILSVRC/Data/ --gpu 0`
the output is:
Epoch: [0][    0/40037]	Time  1.073 ( 1.073)	Data  0.646 ( 0.646)	Loss 6.9085e+00 (6.9085e+00)	Acc@1   0.00 (  0.00)	Acc@5   0.00 (  0.00)
Epoch: [0][   10/40037]	Time  0.018 ( 0.114)	Data  0.000 ( 0.059)	Loss 2.9971e+00 (6.2409e+00)	Acc@1 100.00 ( 81.82)	Acc@5 100.00 ( 81.82)
Epoch: [0][   20/40037]	Time  0.018 ( 0.069)	Data  0.000 ( 0.031)	Loss 0.0000e+00 (3.2690e+00)	Acc@1 100.00 ( 90.48)	Acc@5 100.00 ( 90.48)
Epoch: [0][   30/40037]	Time  0.018 ( 0.053)	Data  0.000 ( 0.022)	Loss 0.0000e+00 (2.2145e+00)	Acc@1 100.00 ( 93.55)	Acc@5 100.00 ( 93.55)
Epoch: [0][   40/40037]	Time  0.018 ( 0.046)	Data  0.000 ( 0.019)	Loss 0.0000e+00 (1.6744e+00)	Acc@1 100.00 ( 95.12)	Acc@5 100.00 ( 95.12)

For some reason the accuracy is perfect after the first batch and loss goes to zero forever. This is the same with 
`python3 main.py -a resnet50 -j 10 --epochs 100 -b 32 --lr 0.1 /data/ImageNet/ILSVRC/Data/ --gpu 0`

Help would be greatly appreciated!! I am hopelessly stuck.

**Comment by codeprb** [comment]
Were you able to find any solution for this @SBoulanger ? I also have the same problem.

**Comment by SBoulanger** [comment]
@codeprb Is your dataset folder structure clean? Make sure there are no unnecessary folders, zip files, floating files.

**Comment by SBoulanger** [comment]
@codeprb If it is, check your version compatibility between cuda and pytorch.
